Gruntfile.js

```diff
@@ -167,6 +167,13 @@ module.exports = function(grunt) {
                 expand: true,
                 flatten: true
             },
+            bootOverCss: {
+                cwd: scgDirPath + 'static/components/css/',
+                src: 'bootstrap-override.css',
```

Comment:
You added file in 2 places, is it really necessary? We don't need to support both files, only one should be committed and added in this file